src/triage/cli.py

```diff
@@ -399,6 +400,37 @@ def __call__(self, args):
         run_crosstabs(db_engine, config)
 
 
+@Triage.register
+class Risklist(Command):
+    """Generate a list of risk scores from an already-trained model and new data"""
+
+    def __init__(self, parser):
+        parser.add_argument(
+            "model_id",
+            type=natural_number,
+            help="The model_id of an existing trained model in the models table",
+        )
+        parser.add_argument(
+            "as_of_date",
+            type=valid_date,
+            help="The date as of which to run features. Format YYYY-MM-DD",
+        )
```

Comment:
Should this be able to handle timestamps rather than just dates (I think the only place we currently assume day-level resolution at the moment is timechop)?

Reply:
Makes sense. It looks like some of the same util code that is pinning timechop to day-level resolution (triage.util.conf.dt_from_str) is also used in risklist. So we could make dt_from_str attempt to read ISO datetime strings as well, which would help in both places.

And replacing the valid_date helper defined here in the CLI module with the improved dt_from_str would help too, since the two functions really do the same thing.

Reply:
That makes sense to me -- is it worth doing that here or as a separate PR?